src/components/Column/ColumnBody.tsx

```diff
@@ -33,58 +34,39 @@ export function ColumnBody({ tickets, onEditClick }: ColumnBodyProps): JSX.Eleme
   );
 }
 
-export function DnDColumnBody({ title, tickets, setTickets, onEditClick }: DnDColumnBodyProps): JSX.Element {
-  function reorder(list: TicketDefinition[], startIndex: number, endIndex: number): TicketDefinition[] {
-    const result = Array.from(list);
-    const [removed] = result.splice(startIndex, 1);
-    result.splice(endIndex, 0, removed);
-
-    return result;
-  }
-
-  const onDragEnd = ({ destination, source }: DropResult) => {
-    if (!destination) {
-      return;
-    }
-
-    const items = reorder(tickets, source.index, destination.index);
-
-    setTickets(items);
-  };
+export function DnDColumnBody({ title, columnId, tickets, onEditClick }: DnDColumnBodyProps): JSX.Element {
+  const grabbedItemContext = useContext(GrabbedItemContext);
 
   return (
-    <DragDropContext onDragEnd={onDragEnd}>
-      <Droppable droppableId="droppable" direction="vertical">
```

Comment:
`DragDropContext` doesn't need to be made on every single ticket in order to move time to another column, so I removed them. Now `<DnDColumnList />` is the central place of DnD logic for components.